resources/migrations/001_update_migrations.yaml

```diff
@@ -8191,6 +8191,50 @@ databaseChangeLog:
               column:
                 name: dashboard_id
 
+  # last_used_at is used to determine if a report card is stale and should be cleaned up
+  # if the card is imported with serialization, we don't want to automatically clean up
+  # newly imported cards, and we don't want to have to make it nullable.
+  - changeSet:
+      id: v50.2024-07-09T20:04:01
+      author: calherries
+      comment: Populate default value for report_card.last_used_at
+      changes:
+        - sql:
+            sql: >-
+              UPDATE report_card
+              SET last_used_at = current_timestamp
+              WHERE last_used_at IS NULL;
```

Comment:
@calherries this condition means that if we started populating `last_used_at` in v50.1 and stopped on v50.7 and then restart again on v50.x, we won't update the values that were set on v50.1. Might we want to change this for the backport to remove this condition entirely, and set _all_ `last_used_at` to `NULL` even if they've been previously set?

For it to be a big issue in practice someone would have to be on v50.1 for a little while, then upgrade to v50.7 for months.

Reply:
I think you mean let's set all `last_used_at` to `current_timestamp`. But that's a great call. I'll create a new PR.

Reply:
Oops, yep - `current_timestamp`. Thanks! 🎉

Reply:
Here's the PR: https://github.com/metabase/metabase/pull/45448